spec/controllers/networks_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe NetworksController, type: :controller, skip: true do
+  let(:current_user) { Fabricate(:admin) }
+
+  before { controller.send :sign_in, current_user }
+
+  def valid_attributes
+    {
+      name: 'Rust'
+    }
+  end
+
+  describe 'Create network' do
+    describe 'with valid attributes' do
+      it 'creates a network and adds to tags' do
+        expect do
+          post :create, { network: valid_attributes}, valid_session
+        end.to change(Tag, :count).by(1)
+      end
+    end
+  end
+end
```

Comment:
add a new line in the end of this file. 
remove the skip: true.
squash commits.